src/app/files/fileManager.js

```diff
@@ -114,10 +114,7 @@ class FileManager extends FileSystemApi {
   }
 
   async getCurrentFile () {
-    const path = this.currentFile()
-    if (!path) throw new Error('no file selected')
```

Comment:
have you checked if catching the error was not handled somewhere? removing this might change the behavior of who is using `getCurrentFile`

Reply:
oups, I forget to add it back. It's corrected now.